docker/test/integration/minifi/__init__.py

```diff
@@ -664,8 +674,8 @@ def nifi_flow_xml(connectable, nifi_version=None, root=None, visited=None):
         input_port_max_concurrent_tasks = Element('maxConcurrentTasks')
         input_port_max_concurrent_tasks.text = '1'
         input_port.append(input_port_max_concurrent_tasks)
-
-        res.iterfind('rootGroup').next().append(input_port)
+        next( res.iterfind('rootGroup') ).append(input_port)
+        """ res.iterfind('rootGroup').next().append(input_port) """
```

Comment:
Guess these are meant to be removed.

Reply:
Yeah it created some invalid code for this. Thanks for pointing this out!